services/templeton/src/templeton/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,86 @@
+package elasticsearch
+
+import (
+	"logcabin"
+
+	"encoding/json"
+	"github.com/mattbaird/elastigo/lib"
+
+	"templeton/database"
+	"templeton/model"
+)
+
+var (
+	logger = logcabin.New()
+)
+
+// Elasticer is a type used to interact with Elasticsearch
+type Elasticer struct {
+	es      *elastigo.Conn
+	baseURL string
+	index   string
+}
+
+// NewElasticer returns a pointer to an Elasticer instance that has already tested its connection
+// by making a WaitForStatus call to the configured Elasticsearch cluster
+func NewElasticer(elasticsearchBase string, elasticsearchIndex string) (*Elasticer, error) {
+	c := elastigo.NewConn()
+
+	err := c.SetFromUrl(elasticsearchBase)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Yeah, Go doesn't shy away from this sort of repetition. You could possibly shorten this chunk a bit like so:

```
    if err := c.SetFromURL(elasticsearchBase); err != nil {
        return nil, err
    }
```
The amount of space it saves is so little that it doesn't really matter which way you do it, IMO.